scripts/js/lib/markdownImages.ts

```diff
@@ -43,7 +43,7 @@ export async function collectInvalidImageErrors(
             ".avif",
           );
           imagesErrors.add(
-            `Convert '${imageName}' to AVIF. You can use the command \`magick public${node.url} public${urlWithAvifExtension}\`. ` +
+            `Convert '${imageName}' to AVIF. You can use the command \`magick public/docs${node.url} public/docs${urlWithAvifExtension}\`. ` +
```

Comment:
I don't think we'll be able to give this precise command in the future when we add Learning